packages/context-menu/src/vaadin-contextmenu-items-mixin.js

```diff
@@ -39,7 +40,7 @@ export const ItemsMixin = (superClass) =>
          * contextMenu.items = [
          *   { text: 'Menu Item 1', theme: 'primary', children:
          *     [
-         *       { text: 'Menu Item 1-1', checked: true },
+         *       { text: 'Menu Item 1-1', checked: true, keepOpen: true },
```

Comment:
Regarding API naming: `keepOpen` seems OK. Some alternatives could be:
- `preventClose`
- `preventAutoClose`
- `autoClose` (requires inverting the value, which could be confusing with no default value)